compiler-rt/test/hwasan/TestCases/libc_thread_freeres.c

```diff
@@ -1,4 +1,4 @@
-// RUN: %clang_hwasan %s -o %t && %env_hwasan_opts=random_tags=1 %run %t
+// RUN: %clang_hwasan -Wno-error=return-type %s -o %t && %env_hwasan_opts=random_tags=1 %run %t
```

Comment:
why not just to return nullptr?

Reply:
It should be fine to change HWASAN tests, it does nothing specific to returning value.